### PR TITLE
added folder logs, and changed log4j2.xml configuration

### PR DESCRIPTION
**Pull request by RomanHreits**

### CODE REVIEW

**Review by IrynaDim — COMMENTED** [review]
At QA, we were told that we could also add an info logger after transaction.commit or transaction.update in our DAOs. To show that our entity was created or updated.

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-        </Logger>
-        <Root level="info" additivity="false">
-            <appender-ref ref="console" />
+        <Root level="all">
+            <AppenderRef ref="Console" level="info" />
+            <AppenderRef ref="File" level="warn" />
         </Root>
     </Loggers>
-</Configuration>
+</Configuration>
```
Comment (KseniiaMakarova) [inline src/main/resources/log4j2.xml]: empty line

File: src/main/java/com/cinema/Main.java [hunk 2]
```diff
@@ -17,10 +17,10 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.List;
-import lombok.extern.log4j.Log4j2;
+import org.jboss.logging.Logger;
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: is this from a different logging dependency that is currently present in your pom.xml?

File: src/main/java/com/cinema/Main.java [hunk 3]
```diff
@@ -118,7 +118,7 @@ public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService = (AuthenticationService) injector
                 .getInstance(AuthenticationService.class);
         authenticationService.register("roman@mail.ru", "roman");
-        log.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
+        logger.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: let's wrap this in try-catch and add log.warn with the informative message in `catch` block

File: src/main/java/com/cinema/Main.java [hunk 4]
```diff
@@ -134,12 +134,12 @@ public static void main(String[] args) throws AuthenticationException {
         //-----------------------------------------------------------------------------------
         OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
         List<Ticket> romanTickets = cartService.getByUser(roman).getTickets();
-        log.info("Shopping cart: " + cartService.getByUser(roman));
-        orderService.completeOrder(romanTickets, roman);
-        log.info("Shopping cart: " + cartService.getByUser(roman));
+        logger.info("Shopping cart: " + cartService.getByUser(roman));
+        logger.info("Roman's order: " + orderService.completeOrder(romanTickets, roman));
+        logger.warn("Shopping cart: " + cartService.getByUser(roman));
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: why warn?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
I agree with Irina on her comment

File: src/main/java/com/cinema/Main.java (hunk 4 above)
Comment (RomanHreits) [reply src/main/java/com/cinema/Main.java]: Я просто грався із кольорами для різних рівнів логування і забув змінити назад

File: src/main/java/com/cinema/Main.java [hunk 5]
```diff
@@ -118,8 +119,11 @@ public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService = (AuthenticationService) injector
                 .getInstance(AuthenticationService.class);
         authenticationService.register("roman@mail.ru", "roman");
-        logger.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
-
+        try {
+            authenticationService.login("roman@mail.ru", "roman");
+        } catch (AuthenticationException e) {
+            logger.error("Login error. Incorrect userEmail or password!", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: error is too much, I think
nothing really bad happened - `warn` level will be enough

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/com/cinema/Main.java (hunk 5 above)
Comment (RomanHreits) [reply src/main/java/com/cinema/Main.java]: done

**Pull request merged**